debian/copyright

```diff
@@ -0,0 +1,22 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: neofs-node
+Upstream-Contact: tech@nspcc.ru
+Source: https://github.com/nspcc-dev/neofs-node
+
+Files: *
+Copyright: 2018-2022 NeoSPCC (@nspcc-dev), contributors of neofs-node project 
+           (https://github.com/nspcc-dev/neofs-node/blob/master/CREDITS.md)
+
+License: GPL-3+
```

Comment:
Should be SPDX compatible
https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/#license-short-name